lms/static/sass/views/_teams.scss

```diff
@@ -148,6 +149,200 @@
     @extend %t-weight4;
   }
 
+  .search-tools {
+    @extend %t-copy-sub1;
+    margin: ($baseline/10) $baseline;
+    color: $gray-l1;
+
+    p, span { // override
+      color: inherit;
+    }
+  }
+
+  .card {
+    @extend .clearfix;
+    position: relative;
+    display: inline-block;
+    width: 30%;
+    margin: ($baseline) 1.2%;
+    border: 1px solid $gray-l4;
+    border-bottom: 2px solid $gray-l3;
+    border-radius: 3px;
+    background-color: $gray-l5;
+    color: $black;
+
+    &:hover,
```

Comment:
NIT: add a "STATE: [states listed here]" comment to the top of these to match our general style doc standards.